configure.ac

```diff
@@ -2,6 +2,7 @@ dnl Process this file with autoconf to produce a configure script.
 
 AC_PREREQ(2.57)
 AC_INIT([libfabric], [0.0.2], [ofiwg@lists.openfabrics.org])
+AC_SUBST(RPM_VERSION, AC_PACKAGE_VERSION)
```

Comment:
Also, I'm using PACKAGE_VERSION instead of version since PACKAGE_VERSION is actually generated by autoconf itself and has helper macros to query its value later in the configure script, etc.: https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.69/html_node/Initializing-configure.html#Initializing-configure

VERSION is generated by automake from back in the days when the package name and version were passed to `AM_INIT_AUTOMAKE`.  So to me it appears that VERSION is legacy and PACKAGE_VERSION should be preferred, unless you can point me at documentation that implies otherwise.